lib/vagrant-libvirt/action/create_network_interfaces.rb

```diff
@@ -194,7 +194,7 @@ def call(env)
               # It's used for provisioning and it has to be available during provisioning,
               # ifdown command is not acceptable here.
               next if slot_number.zero?
-              next if options[:auto_config] === false
+              next if !options[:auto_config]
```

Comment:
I missed that this is actually incorrect, was picked up by the integration test for the private network luckily.

The reason is that it will skip when auto_config is falsey, when we only want to skip when explicitly disabled.

Reply:
Ok, sure. If test fails, I could return a false value for this variable and we check it again. I've answered about this in a PR #1038

Reply:
I've reverted this piece for master as I think the original is probably the best approach, unless there is something specific around the original behaviour that was broken

Reply:
Ok good, I got it! Thanks!